site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
-technologies such as HTML, JavaScript, and CSS.
+This page provides a very brief introduction to what extensions are and how you can make a trivial one.
 
-An extension must fulfill a [single purpose][1] that is narrowly defined and easy to understand. A
+
+## About extensions
+
+Extensions are small software programs that customize the browsing experience. They let users tailor
+Chrome functionality and behavior in many ways, providing things like:
+
+* Productivity tools
+* Web page content enrichment
+* Information aggregation
+* Fun and games
+
+These are just a few examples of the many things that extensions can do. See the [Chrome Web
+Store][5] to see thousands of different examples of published extensions.
+
+### How do extensions work?
+
+Extensions are built on web technologies such as HTML, JavaScript, and CSS.  They run in a separate,
+sandboxed execution environment and interact with the Chrome browser.  Consider the context for this
+-- Chrome runs on a device, connecting the user to resources on the Web, as depicted in this context
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
+
+Also shown in the diagram is an extension "plugged into" the Chrome browser. The extension can
+access and modify selected resources and actions in Chrome. There are two aspects to consider:
+
+**The extensions mechanism**&emsp;This provides a consistent way for users to install and activate
+extensions.
+
+**Extensions APIs**&emsp;The [extensions APIs](/docs/extensions/reference/) allow the extension's
+code to access features of the browser itself, activating tabs, modifying net requests, and so on.
+
+To create an extension, you assemble some resources -- JavaScript and HTML files, a manifest,
+images, and others -- that constitute the extension. For development and testing, you can load these
+"unpacked" into Chrome using extension developer mode. Once you are happy with your extension, you
+can package it and distribute it to users.
+
+### How do users get extensions?
+
+Most Chrome users get extensions from the Chrome Web Store (CWS). Developers across the globe
+publish their extensions in the Chrome Web Store where they are reviewed and made available to end
+users.
+
+{% Aside %}
+
+Some organizations use enterprise policies to install extensions on their user's devices. These
+extensions may either be fetched from the Chrome Web Store or hosted on the organization's web servers.
+{# TODO: See XXX for info on the capabilities and limitations of self hosting. #}
+
+{% endAside %}
+
+You can distribute your extensions through the [Chrome Developer Dashboard][4], publishing them to the [Chrome
+Web Store][5]. For more information, see the Chrome Web Store [developer documentation][6].
+
+### A note about extensions policy
+
+Extensions on the Chrome Web Store must adhere to the [Chrome Web Store policies][cws-policies].
```

Comment:
nit: Can we use a consistent link style?  (I'd lean towards [named-style] instead of [numbered-style], since it's much more clear IMO)

Reply:
The main reason we currently use numbers is that our automated tooling assigned numbers when we ported content over. I don't have any objection to normalizing on named-style. If we do that I'd favor sorting links in the footer alphabetically.

Reply:
Yep, and I don't think we need to go on a spree updating each individual page.  But if we're heavily modifying this one already, I think we should fix it up while we're here. : )

Reply:
And I often like [inline-style](...) ;-)

I have updated the links in this doc to be all [named-style] and alphabetized the definitions part.